project/api/views/stv_view.py

```diff
@@ -154,6 +154,12 @@ def create(self, request, *args, **kwargs):
         Solve overlaps if included in request body
         """
 
+        # Validate other data before overlaps
+        empty_territory_data = request.data.copy()
+        empty_territory_data["territory"] = "POINT (0 0)"
```

Comment:
Can you double check, please, I had an error in `update` method, when visual_center was not a valid geometry
`SRID=4326;POINT(1.2 1.8)` raised an error. Will it cause troubles in create method?

Probably we need to make it a static Point as well.